packages/office-ui-fabric-react/src/components/Callout/examples/Callout.Cover.Example.tsx

```diff
@@ -24,6 +32,63 @@ const DIRECTION_OPTIONS = [
   { key: DirectionalHint.rightBottomEdge, text: 'Right Bottom Edge' }
 ];
 
+const theme = getTheme();
+const styles = mergeStyleSets({
+  buttonArea: {
+    verticalAlign: 'top',
+    display: 'inline-block',
+    textAlign: 'center'
```

Comment:
Was there a reason for getting rid of the `margin: 0 100px` and `min-width: 130px` from `.ms-CalloutCoverExample-buttonArea`?

Reply:
Fixed!